src/functions.php

```diff
@@ -25,3 +28,20 @@ function synchronized(Semaphore $semaphore, \Closure $synchronized, mixed ...$ar
         $lock->release();
     }
 }
+
+/**
+ * @template TReceive
+ * @template TSend
```

Comment:
These can't be inferred, so should we add pseudo class-string arguments to infer the types?

Reply:
I just dropped the template types on the function since they aren't useful. Channels will have to be annotated if someone actually cares to limit it anyway.